packages/alfa-css/src/value/ident.ts

```diff
@@ -0,0 +1,67 @@
+import type { Hash } from "@siteimprove/alfa-hash";
```

Comment:
Good points 😅 
`textual` would make the most sense, as it mimics the CSS grouping. `url` should indeed be moved there too, it is very ad-hoc that it was put in `image` (just because this is the only super-value using it in Alfa…)